src/main/resources/theme/style.css

```
@@ -141,6 +141,30 @@
   -fx-border-color: -fx-box-border;
 }
 
+/***************** Unit Database External *****************/
+
+.unit-database-overlay {
+  -fx-background-color: rgba(0, 0, 0, 0.75);
```

Biome (2.4.4)

[error] Unknown property is not allowed.

(lint/correctness/noUnknownProperty)